Clowns-Science/prototypes/functions.lua

```diff
@@ -1,6 +1,6 @@
 if not mods["angelsrefining"] then --use angels function if available, if not, use this version
-  angelsmods={}
-  angelsmods.functions={}
+  angelsmods = angelsmods or {}
```

Comment:
This block should only activate in the event that angels refining (the library) does not exist. Then again, it may just be easier to have referenced it properly to begin with.

Reply:
On further inspection, refining is required, so that was silly to keep...

Reply:
This PR only touches Clowns-Processing. The change you are commenting on here was from #41. Clown-Science has Angel's Refining as an optional dependency. I believe this change has already been released.